biowasm.json

```
@@ -356,6 +356,22 @@
 				}
 			]
 		},
+		{
+			"name": "hyphy",
+			"programs": ["hyphy"],
+			"description": "HyPhy",
+			"files": ["js", "wasm", "data"],
+			"versions": [
+				{
+					"version": "1.0.9",
+					"branch": "v1.0.9"
+				},
+				{
+					"version": "1.0.11",
+					"branch": "v.1.0.11"
+				}
+			]
+		},
```

This is so biowasm knows which branch to use to fetch which version of Hyphy
```suggestion
			"versions": [
				{
					"version": "2.5.52",
					"branch": "2.5.52"
				}
			]
		},
```